config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml

```diff
@@ -0,0 +1,37 @@
+postsubmits:
+  kubernetes-sigs/node-feature-discovery:
+  - name: postsubmit-node-feature-discovery-e2e-test
+    always_run: true
+    max_concurrency: 1
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-discovery
+      testgrid-tab-name: e2e-test-image
+      description: "Run end-to-end tests for node-feature-discovery after merging a pull request"
+    spec:
+      containers:
+      - image: golang:1.14
+        command:
+        - bash
+        args:
+        - c
+        - apt update && apt install -y jq && scripts/test-infra/test-e2e.sh
```

Comment:
isn't it a cleaner way of doing this?

Reply:
The `apt` commands could be added to the `test-e2e.sh` script, but, I somehow find it cleaner just to hack them here in the job config.

Reply:
In any case, I tried to keep the test script platform/OS agnostic and as simple as possible.